scripts/annotations_import.py

```diff
@@ -0,0 +1,130 @@
+import csv
+from docopt import docopt
+import logging
+import os
+import re
+from tqdm import tqdm
+from web_monitoring import db
+from web_monitoring.db import Client
+
+logger = logging.getLogger(__name__)
+log_level = os.getenv('LOG_LEVEL', 'WARNING')
+logger.setLevel(logging.__dict__[log_level])
+
+class DictReaderStrip(csv.DictReader):
+    @property
+    def fieldnames(self):
+        return [name.strip() for name in super().fieldnames]
+
+def read_csv(csv_path):
+    with open(csv_path, newline='') as csvfile:
+        reader = DictReaderStrip(csvfile)
+        for row in reader:
+            yield row
+
+DIFF_URL_REGEX = re.compile('^.*\/page\/(.*)/(.*)\.\.(.*)')
+def find_change_ids(csv_row):
+    diff_url = csv_row['Last Two - Side by Side']
+    regex_result = DIFF_URL_REGEX.match(diff_url)
+    if regex_result:
+        (page_id, from_version_id, to_version_id) = regex_result.groups()
+        return {'page_id': page_id,
+                'from_version_id': from_version_id,
+                'to_version_id': to_version_id}
+    else:
+        return None
+
+def create_annotation(csv_row, is_important_changes):
+    # Missing step: capture info from the "Who Found This" column and map it to
+    # the author's user record in the database eventually, but that requires a
+    # change to the API
+
+    bool_keys = [
+        'Language alteration',
```

Comment:
This is looking pretty good to me, but the one thing I’m wondering is whether we should attempt to map these names to something a little less prone to change, e.g. “Content change/addition/removal” → “content_changed”.

I’m not sure how often the titles of these columns has been altered without changing their meaning in the past, but I’m fairly sure it’s happened a few times (fixing typos, slight rewordings, etc.). Even though doing a mapping like this means any display needs to also know what text to show, it seems like it would make a more reliable system.

Reply:
Done, please take a look to see if the structure looks alright, and if you'd like to change any of the new json keys